src/components/buttons/button-link-edit.jsx

```diff
@@ -442,13 +442,15 @@ class ButtonLinkEdit extends React.Component {
 		const linkUtils = new CKEDITOR.Link(editor, {
 			appendProtocol: this.props.appendProtocol,
 		});
-		const linkAttrs = {
+		let linkAttrs = {
 			target: this.state.linkTarget,
 		};
```

Comment:
We can use 

```javascript
let linkAttrs = {
  target: this.state.linkTarget || null,
};
```

To avoid the `if` on line `452`

```javascript
   if (!this.state.linkTarget) linkAttrs.target = null;
```

So I'll do that and resend